projects/RabbitMQ.Client/client/impl/AutorecoveringModel.cs

```diff
@@ -297,177 +296,6 @@ private void Dispose(bool disposing)
             }
         }
 
-        public void ConnectionTuneOk(ushort channelMax,
```

Comment:
These were all never actually called

Reply:
Not every method is called by the client, some are received from the server. It's weird to see this particular one on a channel since `connection.tune` is an early connection negotiation step. So I suspect it genuinely was not used.

Reply:
On that note, in one of my PRs I simply removed the negotiation methods from the public facing API since they are just called during negotiation and should never be called explicitly. Something to consider for 7.0.0.